src/nvim/meta_rows.h

```diff
@@ -0,0 +1,9 @@
+#ifndef NVIM_META_ROWS_H
+#define NVIM_META_ROWS_H
+
+int default_status_row(win_T *);
```

Comment:
this is not needed as we use autogenerated headers, just include the `.h` file for the `.c` file that contains the functions `nvim/ex_getln.h` in this case.